src/sentry/conf/server.py

```diff
@@ -1354,6 +1354,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "organizations:discover-basic": True,
     # Enable discover 2 custom queries and saved queries
     "organizations:discover-query": True,
+    # Enables data secrecy mode
```

Comment:
Can we expand on this more? What is data secrecy mode. Imagine you’re a user scrolling g through this list and you were like “what is that”